README.md

```
@@ -2,313 +2,1435 @@
-* [Gabriel Romualdo](https://xtrp.io/) ([@xtrp](https://github.com/xtrp))
-* [Gabriela Radu](http://gabrielaradu.me)
-* [Gabriele Corti](https://borntofrappe.github.io/)
-* [Genesis Gabiola](https://genesisgabiola.netlify.app/)
-* [Georgi Yanev](https://gyanev.com)
-* [Gherciu Gheorghe](https://gherciu.github.io/portfolio/)
-* [Gianluca Fiore](http://gianlucafiore.it)
-* [Gift Egwuenu](https://giftegwuenu.com)
+
+- [Gabriel Carvalho](https://gabrielcarvalho.vercel.app) ([@Cttbiel](https://github.com/cttbiel))
+  [👨🏽‍💻 Computer Engineering Student]
+- [Gabriel Jordan](https://jrdan.dev)[Software Engineer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://jrdan.dev)